src/engine/source/hlp/src/parsers/date.cpp

```diff
@@ -89,8 +89,8 @@ SemParser getSemParser(std::string_view targetField,
  * Supported formats, this will be injected by the config module in due time
  */
 const std::vector<std::tuple<std::string, std::string>> TIME_FORMAT {
-    {"ANSIC", "%a %b %d %T %Y"},        // Mon Jan _2 15:04:05 2006
-    {"UnixDate", "%a %b %d %T %Z %Y"},  // Mon Jan _2 15:04:05 MST 2006
+    {"ANSIC", "%a %b %d %T %Y"},        // Mon Jan 2 15:04:05 2006
+    {"UnixDate", "%a %b %d %T %Z %Y"},  // Mon Jan 2 15:04:05 MST 2006
```

Comment:
Do not delete the `_`, this marks the optional., %d is the day of the month as a decimal number. The modified command %Nd where N is a positive decimal integer specifies the maximum number of characters to read. If not specified, the default is 2. Leading zeroes are permitted but not required.